frontend/src/views/StatsPage.vue

```diff
@@ -1,9 +1,18 @@
 <template>
-  <div id="stats">
-    <h1>Statistiques</h1>
-    <div class="infogram-embed" data-id="d79b1af3-0ab5-4338-a1c9-553d9e0d050e" data-type="interactive" />
-    <h2>👾 Statistiques d’usage et de comportement des personnes utilisatrices de la plateforme</h2>
-    <h3 id="visits">Nombre de visites dans le mois dernier</h3>
+  <div>
+    <h1 class="font-weight-black text-left my-6">Statistiques</h1>
+    <iframe
+      src="http://ma-cantine-metabase.cleverapps.io/public/dashboard/54639f66-d6d0-407c-a9db-766b5fb01e3e"
+      style="border: 1px solid rgb(204, 204, 204);box-shadow: 3px 3px 7px -3px #999;"
+      width="100%"
+      height="400"
+      class="mb-10"
+      allowtransparency
```

Comment:
```suggestion
      allowtransparency
      title="Chartes avec le pourcentage de cantines de chaque type de gestion, et le nombre de nouveaux utilisateurs par jour"
```
Les iframes sont difficile a rendre accessible, je pense qu'on peut merger cette PR et discuter ça après

Reply:
Plus d'iframe :)  Lien vers la page metabase externe : https://github.com/betagouv/ma-cantine/pull/413/commits/e46be6048ea4a642c3de6253cf48b4ff95d2b98b